ui/src/publication/table/publication-table-row.tsx

```diff
@@ -67,7 +67,7 @@ export const PublicationTableRow: React.FC<PublicationTableRowProps> = ({
                         )
                         .join(', ')}
                 </td>
-                <td>{t(`enum.publish-operation.${operation}`)}</td>
+                <td>{t(`enum.Operation.${operation}`)}</td>
```

Comment:
Tuo oli bäkkärillä Operation-nimellä... vaihdoin toistaiseksi näin, mutta Operation on kyllä niin geneerinen että ehkäpä siihen pitäisi suosiolla lisäillä tuo Publish-etuliite kuitenkin

Reply:
Meillä taitaa nytkin jo olla oikeasti useampi erilainen Operation, joten sikäli etuliite voisi olla kiva